rheannemcintosh/london-cocktail-week-2024
# LCW24-4 | Add opening times to the bar dataframe

# [LCW24-4] | Add opening times to the bar dataframe

## Work
This adds the opening times for each of the bars to the dataframe.

## Testing
- Run `jupyter-lab`.
- Run all cells.
- Add an additional cell to display `df`, which should display a list of bars with opening times.

[LCW24-4]: https://rheannemcintosh.atlassian.net/browse/LCW24-4?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
